llvm/tools/llvm-profdata/llvm-profdata.cpp

```diff
@@ -2406,6 +2407,14 @@ static void traverseAllValueSites(const InstrProfRecord &Func, uint32_t VK,
   }
 }
 
+namespace {
+struct InstrProfilePerFuncOptions {
+  bool ShowCounts;
```

Comment:
please initialize scalars at declaration (i.e. `bool ShowCounts = false`), it's very easy to later write code that forgets to initialize a field, and while maybe noticing compiler warnings can highlight this, it's so much easier to just initialize at declaration and avoid undefined behavior!

Reply:
done.